setup.py

```diff
@@ -19,12 +19,12 @@
     packages=find_namespace_packages(include=['asreviewcontrib.*']),
     python_requires='~=3.6',
     install_requires=[
-        'asreview>=0.16',
+        'asreview>=1.0',
     ],
 
     entry_points={
         "asreview.datasets": [
-            "newDataset = asreviewcontrib.dataset_name.your_dataset:YourDataGroup"
+            "example_dataset_group = asreviewcontrib.dataset_name.your_dataset:example_dataset_group", # noqa
```

Comment:
also add remote?

Reply:
The example group has both examples. I think only the group needs to be assigned as entrypoint.